packages/wonder-blocks-form/src/components/text-field.md

```diff
@@ -1,15 +1,91 @@
-A demonstration of a default TextField.
+Text
 
 ```js
 import {TextField} from "@khanacademy/wonder-blocks-form";
 
-<TextField />
+class TextFieldExample extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: "",
+        };
+    }
+
+    render() {
+        return (
+            <TextField
+                id="tf-1"
+                type="text"
+                value={this.state.value}
+                onChange={(newValue) => this.setState({value: newValue})}
+            />
+        );
+    }
+}
+
+<TextFieldExample />
+```
+
+Number
+
+```js
+import {TextField} from "@khanacademy/wonder-blocks-form";
+
+class TextFieldExample extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: "",
```

Comment:
One more thing, could you please add a default value here? so the story shows how the input would look with a number on it.